### ISSUE TITLE
(Suggestion) Hatching Percentage Calculation

### ISSUE THREAD

**Issue by SnakeFang** [GenDeathrow/Hatchery#85]
Line 181 of EggNestTileEntity all uses integer arithmetic and so the percentage is truncated to an integer. If you just make the 100 a float constant (i.e. 100.0F) it will use float arithmetic and won't truncate it. I just thought I'd point it out since the function is supposed to return a float. Happy coding!

**Comment by GenDeathrow** [comment]
Yeah I should have changed that a while back ago. I haven't done it cause well my other code has been so much worst. IE Nesting Pen.  Which I have been rewriting as well as adding in the last of my content. I will have to add it the the list of changes.
